subscription/demo/subscription_demo.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+
+    <record id="subscription_document_partner0" model="subscription.document">
+        <field name="model" ref="base.model_res_partner"/>
+        <field name="name">Partner</field>
+    </record>
+
+    <record id="subscription_document_fields_2" model="subscription.document.fields">
+        <field name="field" ref="base.field_res_partner_name"/>
+        <field name="value">false</field>
+        <field model="subscription.document" name="document_id" search="[('name', '=', u'Partner')]"/>
+    </record>
+
+    <record id="subscription_document_fields_3" model="subscription.document.fields">
+        <field name="field" ref="base.field_res_partner_child_ids"/>
```

Comment:
```suggestion
        <field name="field" ref="base.field_res_partner__child_ids"/>
```

Reply:
Fixed in akretion:12-subscription (but not shown here because the PR was closed in the meantime)